### ISSUE TITLE
macOS VoiceOver navigation does not work on frameless window

### ISSUE THREAD

**Issue by natnan** [electron/electron#20610]
### Preflight Checklist
* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  * 6.0.12
  * 7.0.0-beta.7
* **Operating System:**
  * macOS 10.14.6
* **Last Known Working Electron version:**
  * 5.0.11

### Expected Behavior
macOS VoiceOver to move between elements using caps lock - arrow buttons.

### Actual Behavior
Caps Lock - arrow buttons do nothing while VoiceOver is on.

### To Reproduce
- On Fiddle, add `frame: false` to the `BrowserWindow` options object.
- (Optional) add an element to tab into easily (like `<a> test </a>` into HTML code.
- Run Fiddle and start VoiceOver
- Focus on the window.
- Try to navigate using Caps Lock - arrow buttons. Nothing happens.

You can see what should happen by removing `frame: false` or going back to 5.0.11 electron.

**Comment by qinyangyong** [comment]
5.0.13 also works. Seems 6.0.0 breaks it.

**Comment by electron-triage** [comment]
The Electron version reported on this issue is no longer supported. See our [supported versions documentation](https://www.electronjs.org/docs/tutorial/support#supported-versions).

If this is still reproducible on a supported version, please open a new issue with any other new information that a maintainer should know.

Thank you for taking the time to report this issue and helping to make Electron better! Your help is appreciated.
